src/Service.Tests/SqlTests/GraphQLQueryTests/DwSqlGraphQLQueryTests.cs

```diff
@@ -0,0 +1,304 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Azure.DataApiBuilder.Service.Tests.SqlTests.GraphQLQueryTests
+{
+    /// <summary>
+    /// Test GraphQL Queries validating proper resolver/engine operation.
```

Comment:
Need to have similar tests for REST API GET query support for DW.

Reply:
Can a case with pagination and a case with filter predicate (where foo=bar) be added to? And same for REST.

Reply:
Added file for DW tests. All tests run successfully that are part of the RESTFINDbase class.